apps/web/src/components/invites/MembersTable.tsx

```diff
@@ -83,7 +91,16 @@ export function MembersTable({
                   >
                     Remove Member
                   </DropdownItem>
-                  {canResendInvite(member) ? (
+                  <When truthy={memberInvited(member)}>
+                    <DropdownItem
+                      key="resendInviteBtn"
+                      data-test-id="resend-invite-btn"
```

Comment:
These should probably be changed to "copyInviteBtn" and "copy-invite-btn" 🙂

Reply:
Most definitely is 😁